website/src/docs/components/card/tag/TagCard.js

```diff
@@ -0,0 +1,52 @@
+import Card from '@material-ui/core/Card';
+import CardContent from '@material-ui/core/CardContent';
+import Typography from '@material-ui/core/Typography';
+import RestaurantMenuRounded from '@material-ui/icons/RestaurantMenuRounded';
+import TripOrigin from '@material-ui/icons/TripOrigin';
+import { useTagCardStyles } from '@mui-treasury/styles/cardContent/tag';
+import React from 'react';
+
+const TagCard = () => {
+  const tagStyles = useTagCardStyles();
+  return (
+    <Card className={tagStyles.card} elevation={0}>
+      <CardContent>
+        <TripOrigin className={tagStyles.hole} />
+        <RestaurantMenuRounded
+          fontSize="inherit"
+          style={{ fontSize: '83px', margin: 'auto', width: '100%' }}
+        ></RestaurantMenuRounded>
+        <Typography align="center" gutterBottom>
+          BreakFast
+        </Typography>
+        <Typography
+          color="textSecondary"
+          variant="body2"
+          align="center"
+          component="p"
+        >
+          eggs benedict <br />
+          BLT sandwich <br />
+          french omelette
+        </Typography>
+      </CardContent>
+    </Card>
+  );
+};
+
```

Comment:
Please add
```
// hide-start
...
// hide-end
```
This will remove internal code from code example
![image](https://user-images.githubusercontent.com/18292247/71320002-11e55700-24d8-11ea-8f44-78a13ad48f7e.png)

Reply:
Thank you for checking! I've added those comments. However, still it shows metadata in code sample in my local. Am I missing something or is there any tips to reflect it?

Reply:
I see, you need to delete `/website/.cache` and then `yarn start` again.